src/features/Miscellaneous/AdditionalInformation/index.tsx

```diff
@@ -31,9 +34,16 @@ const AdditionalInformation = ({
 
   const parentRoute = type === 'itgov' ? 'it-governance' : 'trb';
 
-  const isITGovAdmin = useContext(ITGovAdminContext);
```

Comment:
this was not returning the correct value. I logged in as admin, but when logging it out, it returned false.